internal/kafka/internal/presenters/kafka_instance_types.go

```diff
@@ -0,0 +1,23 @@
+package presenters
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/public"
+)
+
+func PresentSupportedKafkaInstanceType(supportedInstanceType *public.SupportedKafkaInstanceType) public.SupportedKafkaInstanceType {
+	reference := PresentReference(supportedInstanceType.Id, supportedInstanceType)
+	return public.SupportedKafkaInstanceType{
+		Id:    reference.Id,
+		Sizes: GetSupportedKafkaSizes(supportedInstanceType.Sizes),
+	}
+}
+
+func GetSupportedKafkaSizes(supportedKafkaSizes []public.SupportedKafkaSize) []public.SupportedKafkaSize {
+	sizes := make([]public.SupportedKafkaSize, 0)
```

Comment:
minor optimization:

We know the size we need for this array so it can be initialized with:
```
 sizes := make([]public.SupportedKafkaSize, len(supportedKafkaSizes))
```

In this way the array won't need to be dynamically re-allocated when iterating

Reply:
Thanks @miguelsorianod. I'll include that